sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java

```diff
@@ -147,9 +150,6 @@ public void stopCollection(final @Nullable String listenerId) {
 
   @SuppressLint("NewApi")
   private void clearCurrentWindow(final @NotNull Window window) {
```

Comment:
```suggestion
  private void stopFrameMetricsTrackingForWindow(final @NotNull Window window) {
```
`m`: since we're not clearing the window in here anymore, I guess it makes sense to rename this method accordingly, how about `stopFrameMetricsTrackingForWindow`?